article/app/controllers/LiveBlogController.scala

```diff
@@ -95,6 +103,25 @@ class LiveBlogController(
 
   // Helper methods
 
+  private def isSupportedTheme(blog: LiveBlogPage): Boolean = {
```

Comment:
For articles I wonder if we have similar types of functions abstracted into ArticlePicker (article/app/services/dotcomponents/ArticlePicker). What do we think about following a similar abstraction for liveblogs (i.e. moving these functions into article/app/services/dotcomponents/LiveblogPicker)? Do we expect to extend the liveblog picking logic over time, or it will only decrease in complexity?

(As an aside, I think ArticlePicker contains the function that chooses the rendering tier. For liveblogs this might be another useful function to abstract into LiveblogPicker, if we think we need LiveblogPicker.)

Reply:
Yeah, we did discuss creating another picker abstraction the same as with articles but hopefully this is as far as the complexity will go for live blogs. We can always break things out later if we need.